src/coffee/card.coffee

```diff
@@ -196,6 +201,19 @@ class Card
     QJ.toggleClass el, @options.classes.valid, test
     QJ.toggleClass el, @options.classes.invalid, !test
 
+  maskCardNumber: (val, el, out) =>
+    mask = @options.masks.cardNumber
+    cardType = payment.fns.cardType(val)
+    numbers = val.split(' ')
+
+    if (cardType == 'amex' && (numbers.length == 3)) || numbers.length >= 4
```

Comment:
Hi Jesse
The amex check isn't really necessary as I dont know much about card numbers and how many groups numbers can exists added amex check to make sure that my code will not break with other card type.
I made a quick check removing the cardType check and works without problems, but again I dont know all the details of card numbers.